open-apparel-registry/open-apparel-registry
# Add FacilityAlias model

## Overview

This tables allows us to keep track of the OAR IDs for facilities that are merged with other facilities or deleted so that we can report on their removal and redirect to their replacement facilities.

We applied the same primary key field settings to `oar_id` that were used for `Facility.id` because we only want a single alias for each OAR ID and we do not want it to change.

Connects #605

## Testing Instructions

* Verify that running `./scripts/update` applies the migration and creates the new table.

## Checklist

- [X] ~CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines~ Skipped because this is an internal change.
